easybuild/easyconfigs/a/augur/augur-7.0.2-intel-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,148 @@
+    }),
+    ('bcbio-gff', '0.6.6', {
+        'modulename': 'BCBio.GFF',
+        'checksums': ['74c6920c91ca18ed9cb872e9471c0be442dad143d8176345917eb1fefc86bc37'],
+    }),
+    ('ConfigArgParse', '1.2.1', {
+        'checksums': ['f30736dcd4e00455ffe3087454799ccb7f9b61d765492dd4b35bbcd62379db12'],
+    }),
+    ('cycler', '0.10.0', {
+        'checksums': ['cd7b2d1018258d7247a71425e9f26463dfb444d411c39569972f4ce586b0c9d8'],
+    }),
+    ('datrie', '0.8.2', {
+        'checksums': ['525b08f638d5cf6115df6ccd818e5a01298cd230b2dac91c8ff2e6499d18765d'],
+    }),
+    ('gitdb', '4.0.4', {
+        'checksums': ['6f0ecd46f99bb4874e5678d628c3a198e2b4ef38daea2756a2bfd8df7dd5c1a5'],
+    }),
+    ('GitPython', '3.1.1', {
+        'modulename': 'git',
+        'checksums': ['6d4f10e2aaad1864bb0f17ec06a2c2831534140e5883c350d58b4e85189dab74'],
+    }),
+    ('importlib-metadata', '1.6.0', {
+        'modulename': 'importlib_metadata',
+        'source_tmpl': 'importlib_metadata-%(version)s.tar.gz',
+        'checksums': ['34513a8a0c4962bc66d35b359558fd8a5e10cd472d37aec5f66858addef32c1e'],
+    }),
+    ('ipython-genutils', '0.2.0', {
+        'modulename': 'ipython_genutils',
+        'source_tmpl': 'ipython_genutils-%(version)s.tar.gz',
+        'checksums': ['eb2e116e75ecef9d4d228fdc66af54269afa26ab4463042e33785b887c628ba8'],
+    }),
+    ('jsonschema', '3.2.0', {
+        'checksums': ['c8a85b28d377cc7737e46e2d9f2b4f44ee3c0e1deac6bf46ddefc7187d30797a'],
+    }),
+    ('jupyter-core', '4.6.3', {
+        'modulename': 'jupyter_core',
+        'source_tmpl': 'jupyter_core-%(version)s.tar.gz',
+        'checksums': ['394fd5dd787e7c8861741880bdf8a00ce39f95de5d18e579c74b882522219e7e'],
+    }),
+    ('kiwisolver', '1.2.0', {
+        'checksums': ['247800260cd38160c362d211dcaf4ed0f7816afb5efe56544748b21d6ad6d17f'],
+    }),
+    ('nbformat', '5.0.5', {
+        'checksums': ['f0c47cf93c505cb943e2f131ef32b8ae869292b5f9f279db2bafb35867923f69'],
+    }),
+    ('packaging', '20.3', {
+        'checksums': ['3c292b474fda1671ec57d46d739d072bfd495a4f51ad01a055121d81e952b7a3'],
+    }),
+    ('phylo-treetime', '0.7.5', {
+        'modulename': 'treetime',
+        'checksums': ['8e3b8003b75ca29b7e383d2ae0eb61dd47a6593eefdbb1e9995d93174b2ed576'],
+    }),
+    ('pyrsistent', '0.16.0', {
+        'checksums': ['28669905fe725965daa16184933676547c5bb40a5153055a8dee2a4bd7933ad3'],
+    }),
+    ('PyYAML', '5.3.1', {
+        'modulename': 'yaml',
+        'checksums': ['b8eac752c5e14d3eca0e6dd9199cd627518cb5ec06add0de9d32baeee6fe645d'],
+    }),
+    ('ratelimiter', '1.2.0.post0', {
+        'checksums': ['5c395dcabdbbde2e5178ef3f89b568a3066454a6ddc223b76473dac22f89b4f7'],
+    }),
+    ('seaborn', '0.9.1', {
+        'checksums': ['da33aa8c20a9a342ce73831d02831a10413f54a05471c7f31edf34f225d456ae'],
+    }),
+    ('smmap', '3.0.2', {
+        'checksums': ['b46d3fc69ba5f367df96d91f8271e8ad667a198d5a28e215a6c3d9acd133a911'],
+    }),
+    ('snakemake', '5.10.0', {
```

Comment:
Since we also have `snakemake` as a stand-alone installation, I would prefer using a proper dep for this too.

May need another patch to the overly restrictive `setup.py`?

Reply:
Latest snakemake is 5.14, but this requires version <5.11 and from changelog it could be `–cores is now a mandatory argument!`
https://snakemake.readthedocs.io/en/stable/project_info/history.html#id17

Reply:
ok, then keep it as extension